opentelemetry-appender-tracing/benches/log-attributes.rs

```diff
@@ -258,14 +258,19 @@ fn criterion_benchmark(c: &mut Criterion) {
 #[cfg(not(target_os = "windows"))]
 criterion_group! {
     name = benches;
-    config = Criterion::default().with_profiler(PProfProfiler::new(100, Output::Flamegraph(None)));
+    config = Criterion::default()
```

Comment:
is there an obvious place we can chuck this to have a "global default" across our crates?

Reply:
Not that I know about. The other thing would be to have the runner specify these on the command line, but there doesn't seem to be any way to configure that in the action.

Reply:
FWIW - I'd like to switch to something like https://bencher.dev/ anyway, but I don't want to scope creep that here. Let's keep in mind that we could factor this out later if we wanted, and leave it as you've done it.